src/core/lib/security/credentials/ssl/ssl_credentials.h

```diff
@@ -120,6 +125,7 @@ class grpc_ssl_server_credentials final : public grpc_server_credentials {
       grpc_ssl_pem_key_cert_pair* pem_key_cert_pairs, size_t num_key_cert_pairs,
       grpc_ssl_client_certificate_request_type client_certificate_request);
 
+  bool send_client_ca_list_ = true;
```

Comment:
I think this field belongs in `grpc_ssl_server_credentials_options` instead of here.

Reply:
Done.